install.sh

```diff
@@ -20,6 +20,10 @@ sudo chown -R $webserver_user:$webserver_group var/logs var/cache
 rm -f web/index_dev.php
 rm -f web/app_dev.php
 
-echo now you need to create your database
-echo afterwards, run php bin/console migrations:migrate
+echo "generate ssh keys"
```

Comment:
What I was trying to say, replace
`echo "generate ssh keys"`
with
`echo "generate ssh keys (set passprhase to talkly, when asked)"`
or something like that.